diff --git a/board/freescale/imx8mm_evk/imx8mm_evk.c b/board/freescale/imx8mm_evk/imx8mm_evk.c
index a2452992151..87fa53f3940 100644
--- a/board/freescale/imx8mm_evk/imx8mm_evk.c
+++ b/board/freescale/imx8mm_evk/imx8mm_evk.c
@@ -22,6 +22,7 @@
 #include <usb.h>
 
 DECLARE_GLOBAL_DATA_PTR;
+extern void early_uart_debug(void);
 
 #define UART_PAD_CTRL	(PAD_CTL_DSE6 | PAD_CTL_FSEL1)
 #define WDOG_PAD_CTRL	(PAD_CTL_DSE6 | PAD_CTL_ODE | PAD_CTL_PUE | PAD_CTL_PE)
@@ -102,6 +103,7 @@ int board_early_init_f(void)
 	setup_gpmi_nand(); /* SPL will call the board_early_init_f */
 #endif
 
+	early_uart_debug(); // Outputs data on all UART ports to see which one you are probing - Spencer Lommel 3/30/25
 	return 0;
 }
 
